.circleci/config.yml

```
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-       - image: circleci/ruby:2.5
+       - image: circleci/ruby:3.0.4
```

We need to use the new `cimg/ruby` images from CircleCI for Ruby 3 compatibility.

```suggestion
       - image: cimg/ruby:3.0.4
```